src/components/Vault/VaultTable.tsx

```diff
@@ -38,7 +37,11 @@ const VaultTable: FC<Pick<VaultProps, "type">> = ({ type }) => {
           <NoVaultsFound />
         ) : (
           vaultAddresses?.map((address, i) => (
-            <VaultRow key={`pool-${i}`} address={address} type={type} />
+            <VaultRow
+              key={`pool-${i}`}
+              address={(address ?? "0x") as `0x${string}`}
```

Comment:
Not necessary. Vault address will never be undefined

Reply:
done